content/nomos/updates/2024-04-08.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 2024-01-08 Nomos weekly
+tags:
+  - nomos-updates
+date: 2024-04-08
+lastmod: 2024-04-08
+draft: false
+description: Weekly update of Nomos
+---
+
+#### `Note`
+
+- This is a weekly update from the part of the team that didn't attend the All-Hands Event in Athens.
+
+## `Mixnet (Network Privacy)`
+
+### `Research`
+
+- Regarding the current situation with the mixnet and staking, we have decided to focus on the simplest approach, that is only single-staking without stake delegation as suggested for the first iteration of the staking. Due to the simplicity of this approach, we can model it more precisely and learn more of its properties.
+- We have received a couple of comments regarding the mixnet from an external reviewer. The comments and our analysis can be seen in this [document](https://docs.google.com/document/d/1yWUP_HdHirwV8XY-i3l_x6qCfh4CQSSa9donAR4AszI/edit).
+- Watched an interesting [critique](https://www.youtube.com/watch?v=1ev4r-aZmFM) of mixnets and suggestions on their improvements. One of them is the Poisson mixing design choice that is part of Nym and is claimed to be wrong. This led to a discussion on how to design a better mixing mechanism. We don't know exactly what is broken in the Poisson mix; there are some claims and some proposals on how to solve the problem suggested in the presentation. We have asked for more paperwork supporting their claims and the authors of the presentation are working on it. In the meantime, a proposal has been prepared based on suggestions from the presentation.
```

Comment:
As this is read by people outside the organization, I don't think a report should include that we or someone "watched a video". The report should focus on pointing out the information of a video and how we think about it.
It's mainly about phrasing, but important for the external presentation.